src/interfaces/patchwork.rs

```diff
@@ -1,11 +1,12 @@
-use super::map::Peer;
+use super::map::{Peer, PeerConnection};
 use super::packet::Packet;
 use std::sync::mpsc::Sender;
 use uuid::Uuid;
 
 pub trait PatchworkState {
     fn new_map(&self, peer: Peer);
     fn route_player_packet(&self, packet: Packet, conn_id: Uuid);
+    fn connect_map(&self, map_index: usize, conn_id: PeerConnection);
```

Comment:
this index thing blows, will be fixed with https://github.com/DuncanUszkay1/Patchwork/issues/118